docs/pages/deployment/discovery.rst

```diff
@@ -35,17 +35,30 @@ If multiple query parameters are specified, all of them must match a single Veri
 Registration
 ============
 
-To register a DID on a Discovery Service, the DID must be activated for the service.
-The Nuts node will then register a Verifiable Presentation of the DID on the service, and periodically refresh it.
-E.g., for service ``coffeecorner`` and DID ``did:web:example.com``:
+To register a subject on a Discovery Service, the subject must be activated for the service.
+The Nuts node will then register a Verifiable Presentation for all subject DIDs on the service, and periodically refresh it.
+E.g., for service ``coffeecorner`` and subject ``example`` (``did:web:example.com``, ``did:nuts:ec6d1834-11e0-48a4-b3f6-934268c7b870``):
 
 .. code-block:: text
 
-    POST /internal/discovery/v1/coffeecorner/did:web:example.com
+    POST /internal/discovery/v1/coffeecorner/example
 
 The DID's wallet must contain the Verifiable Credential(s) that are required by the service definition,
 otherwise registration will fail. If the wallet does not contain the credentials,
-the Nuts node will retry registration periodically.
+the Nuts node will retry registration periodically for all DIDs of a subject.
+
+Optionally, a POST body can be sent which contains `RegistrationParameters`:
+
+.. code-block:: json
+
+    {
+      "registrationParameters": {
+        "endpoint": "https://api.example.com",
+        "contact": "alice@example.com"
+    }
+
```

Comment:
I think it needs an additionele newline after the code block, Github shows it in red

Reply:
no a `}`